tests/tests_unit/graph/test_extractors/test_asset_extractor.py

```diff
@@ -14,9 +14,14 @@ def test_asset_extractor():
 
     store = NeatGraphStore.from_memory_store()
 
-    extractor = AssetsExtractor.from_dataset(client_mock, data_set_external_id="nordic44")
+    extractor = AssetsExtractor.from_dataset(
+        client_mock, data_set_external_id="nordic44", to_type=lambda a: a.metadata.get("type", "Unknown")
```

Comment:
Should `.get` have default fall back to "Asset"?

Reply:
It was just to demonstrate that you can fallback to whatever you would like.